docs/setting-up/client/mongodb.md

```diff
@@ -33,140 +64,213 @@ db.getSiblingDB("admin").createRole({
-which by default is `/etc/mongod.conf`.
+```json
+use admin
+db.setProfilingLevel(2)
+```
 
-In this file, you need to locate the `operationProfiling:` section and add the
-following settings:
+> If you have already [added a service](#add-service), you should remove it and re-add it after changing the profiling level.
 
-```
-operationProfiling:
-   slowOpThresholdMs: 200
-   mode: slowOp
-```
+## Add service
 
-These settings affect `mongod` in the same way as the command line options. Note that the configuration file is in the [YAML](http://yaml.org/spec/) format. In this format the indentation of your lines is important as it defines levels of nesting.
+When you have configured your database server, you can add a MongoDB service with the user interface or on the command line.
 
-Restart the `mongod` service to enable the settings.
+### With the user interface
 
-Run this command as root or by using the `sudo` command
+1. Select *{{icon.cog}} Configuration-->{{icon.inventory}} PMM Inventory-->{{icon.addinstance}} Add Instance*.
+2. Select *MongoDB -- Add a remote instance*.
+3. Enter values for the fields.
+    - *Hostname*: The hostname or IP address of the node where the database server is running.
+    - *Service name*: Choose a service name.
+    - *Port*: Default is 27017 for MongoDB servers.
+    - *Username*: The username for connecting to the MongoDB database. Use the PMM user account if you chose to create one (`{{pmm_mongodb_user}}` in these examples).
+    - *Password*: The password for this user account. (`{{pmm_password}}` in these examples.)
+    - Additional options:
+        - *Use QAN MongoDB Profiler*: Activate if you want to use Query Analytics and have [set up profiling](#profiling).
 
-```sh
-service mongod restart
-```
+4. Click *Add service*.
 
-## Adding MongoDB Service Monitoring
+### On the command line
 
-Add monitoring as follows:
+Use `pmm-admin` to add the database server as a service using one of these example commands.
 
-```sh
-pmm-admin add mongodb --username=pmm --password=pmm
-```
+When successful, PMM Client will print `MongoDB Service added` with the service's ID and name. Use the `--environment` and `-custom-labels` options to set tags for the service to help identify them.
 
-where username and password are credentials for the monitored MongoDB access, which will be used locally on the database host. Additionally, two positional arguments can be appended to the command line flags: a service name to be used by PMM, and a service address. If not specified, they are substituted automatically as `<node>-mongodb` and `127.0.0.1:27017`.
+> **See also**: [`pmm-admin` man page][PMM_ADMIN_MAN_PAGE]
 
-The command line and the output of this command may look as follows:
+**Example**
 
 ```sh
-pmm-admin add mongodb --username=pmm --password=pmm mongo 127.0.0.1:27017
+pmm-admin add mongodb \
+--username={{pmm_mongodb_user}} --password={{pmm_password}} \
+--query-source=profiler --cluster=mycluster
```

Comment:
it is incomplete - it lacks DB address .
the CLI is the following:
```
pmm-admin add mongodb [<flags>] [<name>] [<address>]
    Add MongoDB to monitoring
```

by default 127.0.0.1:27017 is used as address but I would pass it explicitly since not always DB listens this address

Reply:
Good point, thanks, will add.